packages/template-retail-react-app/app/pages/checkout/confirmation.jsx

```diff
@@ -88,18 +104,12 @@ const CheckoutConfirmation = () => {
                 </Fragment>
             )
 
-            const message = /the login is already in use/i.test(error.message)
+            const message = /the login is already in use/i.test(json.detail)
                 ? existingAccountMessage
-                : error.message
+                : json.detail
```

Comment:
The message returned by the API is not guaranteed to be the same locale as the current shopper. The API response should not be displayed.